projects/kit/components/tag/tag-options.ts

```diff
@@ -0,0 +1,36 @@
+import {InjectionToken, ValueProvider} from '@angular/core';
+import {TuiSizeL, TuiSizeS} from '@taiga-ui/core/types';
+import {TuiStatusT} from '@taiga-ui/kit/types';
+
+export interface TuiTagOptions {
+    readonly size: TuiSizeS | TuiSizeL;
+    readonly status: TuiStatusT;
+    readonly hoverable: boolean;
+    readonly removable: boolean;
+    readonly disabled: boolean;
+    readonly autoColor: boolean;
+}
+
+/** Default values for the tag options. */
+export const TUI_TAG_DEFAULT_OPTIONS: TuiTagOptions = {
+    size: 'm',
+    status: 'default',
+    hoverable: false,
```

Comment:
I have doubts about `hoverable`, `removable` and `disabled`, what do you guys think? @splincode @vladimirpotekhin @nsbarsukov 

It looks like it doesn't make sense to change defaults for those since they are very situational and people enable them only in places they need them.

Reply:
Should i go for it and remove these props ?

Reply:
> I have doubts about hoverable, removable and disabled, what do you guys think?

Sorry for the late response!
I agree that all interactive props (`hoverable`, `removable`, `disabled`) should be always `false` by default (without customization by DI-token).

**Argumentation:**
If user set them to true by default (for example, in root `app.module.ts`), it can forget about handlers somewhere deeper in the app. And it will break UX (imagine, tag has removal icon but do nothing when user click on this icon).
I think with explicit setting of this interactive props, there is less probability of this mistake.


@zarghamkhandev is there a need to customize these interactive props (`hoverable`, `removable`, `disabled`) by the token in your project ?  If no, could we temporarily remove them from `TUI_TAG_OPTIONS`. If there is demand on these, we will add them later.

Reply:
removed ✅